dvc/command/run.py

```diff
@@ -99,7 +100,7 @@ def add_parser(subparsers, parent_parser):
         default=[],
         help="Declare dependencies for reproducible cmd.",
         metavar="<path>",
-    )
+    ).complete = completion.FILE
```

Comment:
Could you please confirm/deny that you've tried running completion with this change and it worked for you? :slightly_smiling_face: Thank you!

Reply:
Yes, it works perfectly! :)